### PR TITLE
Config, data source, dashboard for Bitmovin integration

### PR DESCRIPTION
**Pull request by cjmccarthy**
# Summary
Adding a quickstart for our bitmovin integration.

## Pre merge checklist

- [x ] Did you check you NRQL syntax? - Does it work?
- [ x] Did you include a Data source and Documentation reference?
- [x ] Are all documentation links publicly accessible?
- [ x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ x] Did you check your descriptive content for spelling and grammar errors?
- [ x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ x] Does the PR contain a screenshot for each of your dashboards?
- [ x]x Do your screenshots show data?
- [ x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### CODE REVIEW

**Review by RamanaReddy8801 — COMMENTED** [review]
Hi @cjmccarthy, below is the sample template link to use for creating a Quickstart: https://github.com/newrelic/newrelic-quickstarts/tree/main/_template. Please ensure that the content of `config.yml` is specifically related to Bitmovin.

**Comment by cjmccarthy** [comment]
> Hi @cjmccarthy, below is the sample template link to use for creating a Quickstart: https://github.com/newrelic/newrelic-quickstarts/tree/main/_template. Please ensure that the content of `config.yml` is specifically related to Bitmovin.

Thanks for catching that, I had the wrong version of the file submitted. It's fixed now.

**Comment by cjmccarthy** [comment]
Hi @RamanaReddy8801 is this ok to merge?

File: data-sources/bitmovin/config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  Bitmovin empowers businesses to expand their viewer reach and stream video globally with unmatched reliability, scalability, and the highest quality across the broadest range of devices.
+icon: logo.svg
+install:
+  primary:
+    nerdlet: 
+      nerdletId: marketplace.install-data-source
+      nerdletState:
+        dataSourceId: bitmovin-video
+        frameworkConfigId: bitmovin-video
+      requiresAccount: false
```
[the remark this answers is not in the document]
Comment (cjmccarthy) [reply data-sources/bitmovin/config.yml]: Ah, sorry about that. Updated to link to github now, where we have documented the install instructions.

[the remark this answers is not in the document]
Comment (cjmccarthy) [reply data-sources/bitmovin/config.yml]: I have added the logo.svg there as well

**Review by sjyothi54 — APPROVED** [review]
Looks good

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM

**Review by pkudikyala — APPROVED** [review]
LGTM!

**Pull request merged**